synapse/push/baserules.py

```diff
@@ -151,6 +171,16 @@ def make_base_prepend_rules(kind, modified_base_rules):
 ]
 
 
+NEW_PREPEND_OVERRIDE_RULES = [
+    {
+        "rule_id": "global/override/.m.rule.master",
+        "enabled": False,
+        "conditions": [],
+        "actions": [],
+    }
+]
```

Comment:
this is the same as `BASE_PREPEND_OVERRIDE_RULES`, so you might consider dropping it. (nb that "dont_notify" is a no-op)